tests/src/lowering.rs

```diff
@@ -69,6 +73,10 @@ fn test_lowering(fixture: &str) {
         replace_spans(to_ron_string_pretty(&actual_lowered_ast).unwrap()),
     );
 
-    fe_analyzer::analyze(&actual_lowered_ast, src_id)
-        .expect("analysis of the lowered module failed");
+    // TODO: the analyzer rejects lowered nested tuples, because
+    //  nested structs aren't supported yet. we should move the
+    //  not-yet-implemented error to the compiler.
+    //
+    // fe_analyzer::analyze(&actual_lowered_ast, src_id)
```

Comment:
Probably worth creating an issue for it and putting it on the active milestone unless you have that work already in progress anyway.

Reply:
Opened #461